app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from typing import Union
+from app.errors import \
+    NotWearingMaskError,\
+    OutdatedVaccineError,\
+    NotVaccinatedError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> Union[str, None]:
+        try:
+            if datetime.date.today() > visitor["vaccine"]["expiration_date"]:
```

Comment:
use get method instead of parenthesis everywhere to avoid KeyError
like
`visitor.get("vaccine", {}).get("expiration_date")`